### PR TITLE
134 - Top Right Nav

### PR DESCRIPTION
**Pull request by bryansamuel1**

### CODE REVIEW

File: src/components/navigation/TheRoomMenu.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  }, {
+    text: 'Manage Tactics',
+    title: 'Manage your existing tactics within your room'
+  }, {
+    text: 'Manage Room',
+    title: 'Manage the rooms settings'
+  }]
+
+  roomMenuItemsClickHandler (item: RoomMenuItem) {
+    console.log('CLICKED', item)
```
Comment (Eirmas) [inline src/components/navigation/TheRoomMenu.vue]: Remove console log and just disable next line with eslint so pipeline wont fail

File: src/components/navigation/TheUserMenu.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  userMenuItems: UserMenuItem[] = [{
+    text: 'Login',
+    title: 'Login to save your progress'
+  }, {
+    text: 'Logout',
+    title: 'Logout of your logged in account'
+  }]
+
+  userMenuItemsClickHandler (item: UserMenuItem) {
+    console.log('CLICKED', item)
```
Comment (Eirmas) [inline src/components/navigation/TheUserMenu.vue]: Same for this one

File: src/components/navigation/TheRoomMenu.vue (hunk 1 above)
Comment (bryansamuel1) [reply src/components/navigation/TheRoomMenu.vue]: I have removed them. It is outdated now :)

File: src/components/navigation/TheUserMenu.vue (hunk 2 above)
Comment (bryansamuel1) [reply src/components/navigation/TheUserMenu.vue]: I have removed them. It is outdated now :)

**Pull request merged**